services/galley/src/Galley/API/Update.hs

```diff
@@ -980,7 +984,7 @@ handleOtrResponse ::
 handleOtrResponse usr clt rcps membs clts val now go = case checkOtrRecipients usr clt rcps membs clts val now of
   ValidOtrRecipients m r -> go r >> pure (OtrSent m)
   MissingOtrRecipients m -> do
-    guardLegalholdPolicyConflicts usr (missingClients m)
+    guardLegalholdPolicyConflicts (ProtectedUser usr) (missingClients m)
```

Comment:
Are you sure it's always a user here, never a bot?

Reply:
Actually, I think you're right: bots have their own, entirely separate API.  I've always been annoyed by that, but at least this once it works in our favor.  :)